dre-service/pom.xml

```diff
@@ -68,6 +68,11 @@
       <artifactId>cdap-etl-core</artifactId>
       <version>${cdap.version}</version>
     </dependency>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-common</artifactId>
```

Comment:
this is meant to be an internal common module for the CDAP platform. It includes a bunch of dependencies that should not be used in apps.

Reply:
Done